### DISCUSSION TITLE
How to fix This error

### DISCUSSION THREAD

**Discussion by Gray217** [GitSquared/edex-ui#1142] [Questions]
[10550:0722/192826.522396:FATAL:electron_main_delegate.cc(252)] Running as root without --no-sandbox is not supported. See https://crbug.com/638180.
/root/edex-ui/node_modules/electron/dist/electron exited with signal SIGTRAP
npm ERR! code 1
npm ERR! path /root/edex-ui
npm ERR! command failed
npm ERR! command sh -c electron src --nointro "electron" "src"

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-07-22T13_58_26_549Z-debug.log

**Comment by GitSquared** [comment]
Don't run eDEX as root.
